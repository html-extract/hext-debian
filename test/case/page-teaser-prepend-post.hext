<a title:prepend("Post: "):post />
