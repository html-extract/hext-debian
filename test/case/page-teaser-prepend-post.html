<a title="Foo">foo</a>
<a title="Bar">bar</a>
<a title="Foobar">foobar</a>
<a title="Barfoo">barfoo</a>
