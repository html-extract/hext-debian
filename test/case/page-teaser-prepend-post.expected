{"post":"Post: Foo"}
{"post":"Post: Bar"}
{"post":"Post: Foobar"}
{"post":"Post: Barfoo"}
